apps/condo/pages/auth/register.tsx

```diff
@@ -253,10 +247,21 @@ const InputPhoneForm = ({ onFinish }): React.ReactElement<IInputPhoneFormProps>
             onFinally: () => {
                 setIsLoading(false)
             },
+            OnErrorMsg: null,
             intl,
             form,
-            ErrorToFormFieldMsgMapping,
         }).catch(error => {
+            // We expect only 1 type of error here. That's why - we override standard behaviour with OnErrorMsg = null
+            if (has(error, 'graphQLErrors')) {
+                const smsTimeoutError = error.graphQLErrors.find((err) => (err.name.indexOf(TOO_MANY_REQUESTS) === 0))
+                if (smsTimeoutError) {
+                    const { data: { timeRemain = 0 } } = smsTimeoutError
+                    notification.error({
+                        message: SMSTooManyRequestsError,
+                        description: intl.formatMessage({ id: 'pages.auth.register.error.smstimeout' }, { timeRemain }),
+                    })
+                }
+            }
             setIsLoading(false)
```

Comment:
If another kind of error occurs on server, nothing will be displayed to user, because you have suppressed default errors. Tested on my machine by adding `throw new Error('test')` into resolver of `startConfirmPhoneAction` mutation.

Reply:
We don't need to react at every I'm unknown server error.

Reply:
it's better to give feedback about errors. rather than just hide it